### PR TITLE
Fix 176

### PR DESCRIPTION
**Pull request by Alim-faraji**

### CODE REVIEW

**Comment by Alim-faraji** [comment]
[This ](https://github.com/QuantEcon/lecture-datascience.myst/pull/178/commits/792b617a47551f7c7a4a7eddf6d7e37704e47d5c) commit causes failure in executing the codes. It seems like some of the modules are installed along with `qeds`, so when I removed `qeds` from the dependencies, we get `ModuleNotFoundError`. What do you think @wupeifan?

File: lectures/applications/regression.md [hunk 1]
```diff
@@ -263,15 +256,15 @@ one more time -- this time including the prediction from both of our linear mode
 ```{code-cell} python
 ax = var_scatter(df)
 
-def scatter_model(mod, X, ax=None, color=colors[1], x="sqft_living"):
```
Comment (wupeifan) [inline lectures/applications/regression.md:266]: Why not just hardcode `colors` vector and then keep these lines unchanged? You can just do
`colors = ['#165aa7',
 '#cb495c',
 '#fec630',
 '#bb60d5',
 '#f47915',
 '#06ab54',
 '#002070',
 '#b27d12',
 '#007030']`

**Comment by wupeifan** [comment]
LGTM except for the only one comment above. You can merge it after that

**Comment by Alim-faraji** [comment]
Thanks. But I couldn't remove `qeds` line from `environment.yml`. The build failed and I had to revert that commit back.

**Comment by wupeifan** [comment]
Can you do a global search of the md files to see whether there are any other `qeds` calls left? If there's none, then it should be safe to remove it everywhere in this repo

**Comment by wupeifan** [comment]
Maybe we should ask @mmcky about the necessity of  `qeds` in `environment.yml` file? Theoretically it should be fine if we remove all `qeds` dependence from the markdown files, no?

**Comment by mmcky** [comment]
@wupeifan @Alim-faraji if there are not instances of `qeds` being imported it should be able to be removed.

**Comment by wupeifan** [comment]
@mmcky I see. Then it must be `qeds` including a bunch of packages that is otherwise not included, like `seaborn`, `patsy`, `quandl`, etc. The `qeds` package is really wrapping a lot of stuff.
@Alim-faraji Let's left the `qeds` stuff in `environment.yml` in this PR, and in the next PR, we replace with all independent packages. You can see those from the logs of the failed build above. I checked them -- I think all of them are missing packages.

File: lectures/applications/regression.md [hunk 2]
```diff
@@ -408,7 +403,6 @@ alphas, coefs_lasso, _ = linear_model.lasso_path(X, y, alphas=alphas, fit_interc
 
 # plotting
 fig, ax = plt.subplots(figsize=(12, 8))
-colors = cycle(qeds.themes.COLOR_CYCLE)
```
Comment (wupeifan) [inline lectures/applications/regression.md:411]: This line is still needed, I think?

Comment (wupeifan) [reply lectures/applications/regression.md:411]: OK I added this line back with an associated change. If this passes then I think this one can be merged.

**Pull request merged**